src/Action_SetVelocity.cpp

```diff
@@ -3,28 +3,67 @@
 #include "CpptrajStdio.h"
 #include "Constants.h"
 
-Action_SetVelocity::Action_SetVelocity() : tempi_(0.0) {}
+Action_SetVelocity::Action_SetVelocity() : tempi_(0.0), zeroMomentum_(false) {}
 
 void Action_SetVelocity::Help() const {
-  mprintf("\t[<mask>] [tempi <temperature>] [ig <random seed>]\n"
-          "  Set velocities in frame for atoms in <mask> using Maxwellian distribution\n" 
-          "  based on given temperature.\n");
+  mprintf("\t[<mask>] [{tempi <temperature> | modify}] [ig <random seed>]\n"
+          "\t[%s] [%s]\n", Constraints::constraintArgs, Constraints::rattleArgs);
+  mprintf("\t[zeromomentum]\n");
+  mprintf("  Set velocities in frame for atoms in <mask> using Maxwellian distribution\n" 
+          "  based on given temperature; default 300.0 K. If tempi is 0.0 set\n"
+          "  velocities of atoms in mask to 0.0.\n"
+          "  If 'modify' is specified do not set; only modify existing velocities.\n"
```

Comment:
> what is the different between 'set' and 'modify'?

The default mode `set` is for setting velocities from a Maxwell-Boltzmann distribution; this happens whether or not the incoming frame has velocity information or not. The `modify` keyword essentially means do not do this, just modify any existing velocities. This allows `setvelocity` to be used to e.g. zero out the total momentum in a system via `zeromomentum` or to modify velocities to be consistent with constraints via `ntc`.